pkg/cli/flags.go

```diff
@@ -358,6 +359,7 @@ func init() {
 		VarFlag(f, &serverCfg.Stores, cliflags.Store)
 		VarFlag(f, &serverCfg.StorageEngine, cliflags.StorageEngine)
 		VarFlag(f, &serverCfg.MaxOffset, cliflags.MaxOffset)
+		StringFlag(f, &clockDevice, cliflags.ClockDevice, "")
```

Comment:
Was there a reason not to do `StringFlag(f, &serverCfg.ClockDevice, ...)` and avoid the separate global var?

Reply:
I changed it as you are suggesting.